docs/conf.py

```diff
@@ -332,5 +332,7 @@
 intersphinx_mapping = {
     'python': ('https://docs.python.org/2.7/', None),
     'werkzeug': ('http://werkzeug.pocoo.org/docs/', None),
+    # FIXME HTTPS certificate is invalid
     'flask': ('http://flask.pocoo.org/docs/', None),
+    # FIXME HTTPS certificate is invalid
```

Comment:
we usually put FIXME comments above the code that needs to be fixed.
You can join both `pocoo.org` FIXME comments into one: `FIXME HTTPS certificate is not valid for *.pocoo.org`